dockerfiles/Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM continuumio/miniconda3:latest
```

Comment:
Should this perhaps be condaforge/miniforge3?

Reply:
Hi @benjamb
I don't understand the comment. Could you please provide me more details about what you suggest to do?
Thanks!

Reply:
Apologies. I was wondering if we should use `condaforge/miniforge3:latest` as the base image? This is a point for discussion though.

Reply:
Good question, I never used miniforge.

It seems that the miniforge image is based on the Ubuntu bionic image while miniconda3 is based on a Debian one. I guess Debian is lighter.
Does miniforge add anything compared to miniconda?

We should keep a default one for the moment then switch to a mamba or umamba as soon as available don't you think?

Reply:
One benefit is that miniforge installers have conda-forge as the default channel

Reply:
Thanks! I saw that too.

I will dig a bit what's the base layers sizes to keep it as minimale as possible.
That's much more important for pulling time and sparing resources than adding the -c conda-forge or a specific .condarc in the Dockerfile.

Reply:
The miniforge3 image just so happens to be ⅔ the size of the miniconda3 image.

A µmamba image would be very cool though.

Reply:
Thanks you were faster than me ;). Uncompressed, the miniforge3 image is still 2/3 of the miniconda3.
I was surprised the bionic image to be also lighter than the Debian one.

Let's switch to the miniforge3 ! Thanks @benjamb @tom--pollard